.github/workflows/e2e.yml

```diff
@@ -65,6 +65,7 @@ jobs:
       - name: Run end-to-end tests
         env:
           SANITY_E2E_SESSION_TOKEN: ${{ secrets.SANITY_E2E_SESSION_TOKEN }}
+          PWTEST_BLOB_REPORT_NAME: ${{ matrix.project }}
```

Comment:
is there any official docs about `PWTEST_BLOB_REPORT_NAME` we could link to here?

Reply:
Unfortunately there is no docs for it, there is an open issue to update docs for adding the env var to the docs but I will link to the code in the meantime.

Reply:
no worries 👍🏼